device/cuda/src/clusterization/clusterization_algorithm.cu

```diff
@@ -11,13 +11,9 @@
 #include "traccc/cuda/utils/definitions.hpp"
 
 // Project include(s)
-#include "traccc/clusterization/device/connect_components.hpp"
-#include "traccc/clusterization/device/count_cluster_cells.hpp"
-#include "traccc/clusterization/device/create_measurements.hpp"
-#include "traccc/clusterization/device/find_clusters.hpp"
+#include "traccc/clusterization/device/aggregate_cluster.hpp"
```

Comment:
I am going to assume that this is just a copy of the CUDA code in `component_connection.cu`, so I will skip over it.